spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+require 'json'
+
+RSpec.describe UsersController, type: :controller do
+
+    validUserTestParams = { user: {first_name: "UserFirstName", 
```

Comment:
`validUserTestParams` should be snake case

Reply:
This should be nested like this:

```
validUserTestParams = {
  user: {
    first_name: "UserFirstName",
    last_name: "UserLastName", 
    ...
    password_confirmation: "UserPassword"
  }
}

Reply:
fixed